src/appkit.cs

```diff
@@ -2218,6 +2218,18 @@ interface NSButton : NSAccessibilityButton {
 		[Mac (10, 12, 2)]
 		[NullAllowed, Export ("bezelColor", ArgumentSemantic.Copy)]
 		NSColor BezelColor { get; set; }
+
+		[Mac (10,13)]
+		[Export ("compressWithPrioritizedCompressionOptions:")]
+		new void Compress (NSUserInterfaceCompressionOptions [] prioritizedOptions);
```

Comment:
why the `new` on the API ?

Reply:
To silence compiler warnings, those are already defined in the `NSUserInterfaceCompression` protocol that this object implements

Reply:
It won't change generated code since they are already inlined but they are nice to have explicitly in the interface because they will show up and won't appear as "missing" in future reviews.

Reply:
There's some case where we must duplicate (e.g. conflicting availability) but it does not seems to be the case here. 

Even then (if needed) there should be a comment (or region) that mention the (original) protocol and why it was duplicated.

If there's no required changes then let use xtro to catch those :) it's safer and duplicated bindings makes future reviewing harder (introduce mismatches) that will be harder to identify.

Reply:
```csharp
[BaseType (typeof(NSControl))]
interface NSButton : INSUserInterfaceValidations, INSAccessibilityButton, INSUserInterfaceCompression
{
   // omitted code for brevity 

    [Mac (10, 12, 2)]
    [NullAllowed, Export ("bezelColor", ArgumentSemantic.Copy)]
    NSColor BezelColor { get; set; }

   [Mac (10,13)]
    [Export ("compressWithPrioritizedCompressionOptions:")]
    void CompressWithPrioritizedCompressionOptions (NSUserInterfaceCompressionOptions[] prioritizedOptions);

    [Mac (10,13)]
    [Export ("minimumSizeWithPrioritizedCompressionOptions:")]
    CGSize MinimumSizeWithPrioritizedCompressionOptions (NSUserInterfaceCompressionOptions[] prioritizedOptions);

    // @property (readonly, copy) NSUserInterfaceCompressionOptions * _Nonnull activeCompressionOptions __attribute__((availability(macos, introduced=10.13)));
    [Mac (10, 13)]
    [Export ("activeCompressionOptions", ArgumentSemantic.Copy)]
    NSUserInterfaceCompressionOptions ActiveCompressionOptions { get; }
}
```

Reply:
Ok, makes sense, I'll remove them 👍

Reply:
My point is that the diff was a lot larger than what xtro reports - which _hinted_ at something that needed _minimally_ some comments to explain why.

Note that there might be bugs in xtro too :) so comparing diff/xtro is likely our best bet to fix them. E.g. there's no mention of `NSSegmentedControl` missing a conformance